src/sb_pointcloud_processing/launch/line_extractor_rviz_degree_3.launch

```diff
@@ -39,4 +40,4 @@
         <!-- defines how far apart the points are for the outlier line -->
         <param name="outlier_x_delta" value="0.1" type="double"></param>
     </node>
-</launch>
\ No newline at end of file
+</launch>
```

Comment:
there's no need to change this file since this is a launch file specifically for testing rviz, and does not affect the overall pipeline.

Reply:
👌